synfig-studio/src/gui/docks/dock_info.h

```diff
@@ -53,6 +53,9 @@ class Dock_Info : public Dock_CanvasSpecific
 	Gtk::ProgressBar render_progress;
 	Gtk::Label       render_percentage;
 	Gtk::Button      stop_button;
+	Gtk::Button   open_button;
```

Comment:
either align the variable name with the one of previous line or use single space.